pkg/master/ports/ports.go

```diff
@@ -39,7 +39,4 @@ const (
 	// KubeControllerManagerPort is the default port for the controller manager status server.
 	// May be overridden by a flag at startup.
 	KubeControllerManagerPort = 10257
-	// CloudControllerManagerPort is the default port for the cloud controller manager server.
-	// This value may be overridden by a flag at startup.
-	CloudControllerManagerPort = 10258
```

Comment:
I don't think it hurts to leave this here actually.

Reply:
I'd been wondering about "CloudControllerManagerPort = cloudprovider.CloudControllerManagerPort" Most IDEs will give the value on mouse over. It also provides a clue of another reserved port.

Reply:
Added CloudControllerManagerPort back